src/components/utils/ManagedTree.js

```diff
@@ -31,6 +38,16 @@ let ManagedTree = React.createClass({
     }
   },
 
+  expandListItems(treeListItems) {
+    const expanded = this.state.expanded;
+    for (let i = treeListItems.length - 1; i >= 0; i--) {
+      const key = this.props.getKey(treeListItems[i]);
+      expanded.add(key);
+    }
```

Comment:
you can clean this up by doing:

```js
treeListItems.forEach(item => this.state.expanded.add(this.props.getKey(item))
```

Not a big deal, but you know, we're perfectionists here :)